test/integration/attachment.test.js

```diff
@@ -99,17 +99,17 @@ describeAuthOnly('attachments - authenticated', function () {
   it('should return from the octet route', async function () {
     const attachment = '00000000-0000-1000-8000-000000000001'
     const response = await getAttachmentRouteOctet(attachment, app, authToken)
-    expect(response.status).to.equal(200)
+    expect(response.status).to.equal(201)
   })
 
-  it('should return 406 when requesting json from the octet route', async function () {
-    const attachment = '00000000-0000-1000-8000-000000000001'
-    const response = await getAttachmentRouteJSON(attachment, app, authToken)
-    expect(response.status).to.equal(406)
+  it('should return 500 when requesting json from the octet route', async function () {
```

Comment:
this should be a `406`. Have we somehow lost this functionality? I think there is a typo in that this should be 

```
should return 406 when requesting json from the octet attachment
```

Reply:
Yeah my custom errors were largely removed for some reason and 500's chucked in. I'll add mine back